Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/install_hooks.py

```diff
@@ -150,6 +150,21 @@ def main(cls, args, repository, hooks=None, identifier_template=None, **kwargs):
             sys.stderr.write('No hooks to install\n')
             return 1
 
+        url = repository.url()
+        match = repository.SSH_REMOTE.match(url) or repository.HTTP_REMOTE.match(url)
+        if match:
+            path = match.group('path')
+            if 'webkit' not in path.lower():
```

Comment:
You're going to match anything that's `WebKit/<project>` or `<org>/WebKit`. I'm approving this, although we should consider basing this off of a variable in `metadata/git_config_extension` instead, which would be more generic.